common/oidc-client/oidc-authservice/base/kustomization.yaml

```diff
@@ -42,6 +42,6 @@ vars:
 configurations:
   - params.yaml
 images:
-  - name: gcr.io/arrikto/kubeflow/oidc-authservice
```

Comment:
We'll only need to update the `newName` key with the dockerhub registry, and make sure the `name` matches what is initially in the StatefulSet

Reply:
Changed, thanks!